soc/arm/nxp_imx/rt/soc_rt10xx.c

```diff
@@ -224,11 +224,16 @@ static ALWAYS_INLINE void clock_init(void)
 	defined(CONFIG_MEMC_MCUX_FLEXSPI) && \
 	DT_NODE_HAS_STATUS(DT_NODELABEL(flexspi), okay)
 	CLOCK_DisableClock(kCLOCK_FlexSpi);
-	CLOCK_InitUsb1Pfd(kCLOCK_Pfd0, 24);
+#endif
+
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(flexspi), okay) && \
+	defined(CONFIG_MEMC_MCUX_FLEXSPI)
+	CLOCK_InitUsb1Pfd(kCLOCK_Pfd0, 26);
 	CLOCK_SetMux(kCLOCK_FlexspiMux, 3);
-	CLOCK_SetDiv(kCLOCK_FlexspiDiv, 2);
+	CLOCK_SetDiv(kCLOCK_FlexspiDiv, 3);
```

Comment:
All the changes in this file, feel a bit like magic, and can have impact on a lot of boards. These shouldn't be changed without proper testing/confirmation.

Reply:
I had two ideas. One was to move these values to dts instead so each soc had its own settings. 
The other one was just to change it since these values were copied from the IMXRT1050 SDK.

Reply:
I had two ideas. One was to move these values to dts instead so each soc had its own settings. 
The other one was just to change it since these values were copied from the IMXRT1050 SDK.

Reply:
@Kludentwo , those settings are varied depends on what flespi device your are using, so looks like option 1 is better.

Reply:
@hakehuang I have a bad feeling about added more changes to this PR. Most  likely i will remove this commit and create a new PR with the DTS changes.

Reply:
I agree.

Reply:
Removed this for future PR.